### PR TITLE
Add LICENSE and round setup.py

### PR DESCRIPTION
**Pull request by mdellweg**
This is kind of urgent, and i guess i need input from all of you. Sorry for the inconvenience.

@daviddavis
@ggainey 
@bmbouter 
@dkliban

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -3,23 +3,36 @@
...
     },
-    extras_require={
-        "pygments": ["pygments"],
-    },
+    license="GPLv2+",  # TODO CHECK THIS
+    classifiers=[
+        "Development Status :: 3 - Alpha",
+        "Environment :: Console",
+        "Intended Audience :: System Administrators",
+        "License :: OSI Approved :: GNU General Public License v2 or later (GPLv2+)",  # TODO CHECK THIS
```
Comment (mdellweg) [inline setup.py:32]: Once you all agreed, i will remove these comments.

**Review by daviddavis — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (ggainey) [reply setup.py:32]: GPL2+ works for me, I approve

**Review by ggainey — APPROVED** [review]

**Review by bmbouter — APPROVED** [review]
This looks great. Thank you.

**Comment by daviddavis** [comment]
Going to merge this so we can secure pulp-cli on pypi. Will get @dkliban's approval when he returns.

**Pull request merged**